src/cmap/connect.ts

```diff
@@ -231,7 +231,7 @@ export function prepareHandshakeDocument(
   const { serverApi } = authContext.connection;
 
   const handshakeDoc: HandshakeDocument = {
-    [serverApi?.version ? 'hello' : LEGACY_HELLO_COMMAND]: true,
+    [serverApi?.version ? 'hello' : LEGACY_HELLO_COMMAND]: 1,
```

Comment:
Is there a way we can get regression coverage here? It looks like we only have a test that asserts the monitor will use `hello: 1`.

Reply:
Added 2 unit tests for this now.